packages/server/src/script/resendSignupEmails.ts

```diff
@@ -0,0 +1,136 @@
+import * as admin from 'firebase-admin'
+import { FirestoreService } from '../service/firestore'
+import { Letter } from '../service/letter'
+import { RichStateInfo, TwilioResponse } from '../service/types'
+import { sendSignupEmail } from '../service/mg'
+import { storageFileFromId } from '../service/storage'
+import { sendFaxes } from '../service/twilio'
+import { WithId } from '../common'
+import { toContactMethod } from '../common'
+import { toPdfBuffer } from '../service/pdf'
+
+const { input } = process.env.CI ? require('./data/conditions.sample.ts') : require('./data/conditions.nogit.ts')
+
+const firestoreService = new FirestoreService()
+type WhereFilterOp = "<" | "<=" | "==" | ">=" | ">" | "array-contains" | "in" | "array-contains-any"
```

Comment:
Nit: we use two-space tabs.  I'm not sure why you're passing lint here.

Reply:
Do you mean two new lines? Or you think I should have two spaces between the " | " characters?